src/leihs/my/initial_admin/back.clj

```diff
@@ -16,8 +17,11 @@
 
 
 (defn some-admin? [tx]
-  (->> ["SELECT true AS has_admin FROM users WHERE is_admin = true"]
-       (jdbc/query tx ) first :has_admin boolean))
+  (let [query (sql-format {:select [:is_admin]
+                           :from [:users]
+                           :where [:= :is_admin true]})
```

Comment:
prefer the honeysql functions representation anyway, otherwise each keyword on a newline.